build/build.proj

```diff
@@ -170,12 +170,7 @@
           LogStandardErrorAsError="true"
           WorkingDirectory="$(TestsDirectory)"
           IgnoreExitCode="true"
-          >
-      <Output TaskParameter="ExitCode" PropertyName="ExitCode" />
-    </Exec>
-
-    <Error Text="There were test failures, for full log see %(XmlTestFile.FullPath)" Condition="$(ExitCode) != 0" />
-
```

Comment:
Removed it because xunit returns non-zero even when we have skipped tests. Which now that we have skip attributes happened and caused the build to fail. We don't need this anymore because we are parsing the testResult.xml file and jenkins will fail if there are any failed tests in there.